src/components/Panel/Panel.spec.jsx

```diff
@@ -19,6 +19,16 @@ describe('Panel', () => {
 			assert.equal(wrapper.find('.lucid-Panel-header').length, 1);
 		});
 
+		it('passes through className', () => {
+			const wrapper = mount(
```

Comment:
All these tests should be using `shallow` instead of `mount` since none of them make any assertions on the DOM. Also as a rule, when we use `mount`, be sure to use the `describeWithDOM` helper instead of the regular `describe`.